.mailmap

```diff
@@ -1546,6 +1546,7 @@ anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.n
 anutosh491 <andersonbhat491@gmail.com> anutosh491 <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
 atharvParlikar <atharvparlikar@gmail.com>
+bin/mailmap_check.py
```

Comment:
You don't need to include this in `.mailmap`.

Reply:
yes, you're right, sorry I must have put it there by mistake (I'll correct it), thank you!